src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -336,7 +336,99 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     > Learn more about [error handling](#{Guides::HttpAndRouting::ErrorHandling.path}).
 
-    ## Query parameters
+    ## Handling parameters
+
+    Parameters, or `params`, are data that is sent from client back to the server. There are a few different ways this can happen:
+
+    * Path parameters - The dynamic values passed in your route path. e.g. `/users/:id`.
+    * Query parameters - These are added to the end of a URL after the `?` in key/value pairs. e.g. `?page=1`
+    * Form parameters - When you perform an HTTP POST, the data sent through the body. This may be formatted as JSON, or form data
+    * Multipart parameters - Similar to form parameters, but generally to contain a file
+
+    When parameters are passed to an action, you have access to these with the `params` method.
+
+    ```crystal
+    # src/actions/users/index.cr
+    class Users::Index < BrowserAction
+
+      # example.com/users?page=1&filter=active
```

Comment:
Something about this url being inside the example file feels odd to me. How about putting it up in a sentence before the code sample:

> Here's an example of using parameters when visiting the `/users?page=1&filter=active` path: